### PR TITLE
Updated einsteinpy.symbolic Jupyter Notebooks

### LINKED ISSUE #549: Some of `einsteinpy.symbolic` Jupyter Notebooks are outdated
🐞 **Problem**
As the title suggests, some of the Jupyter Notebooks (stored at `einsteinpy\docs\source\examples\`), containing examples for the `symbolic` module are out of date. Compare these (**_OLD vs NEW_**):

1. https://docs.einsteinpy.org/en/latest/examples/Weyl%20Tensor%20symbolic%20calculation.html
image
image

2. https://docs.einsteinpy.org/en/latest/examples/Ricci%20Tensor%20and%20Scalar%20Curvature%20symbolic%20calculation.html
image
image

3. https://docs.einsteinpy.org/en/latest/examples/Einstein%20Tensor%20symbolic%20calculation.html
image
image

(Check other notebooks as well)

The output is the same, except in the old API, mathematical expressions were not completely simplified, while the new API simplifies those, as much as possible.

🎯 **Goal**

To "simplify" the output and make the results in the examples as accurate as they can be.

💡 **Possible solutions**

A simple notebook re-run with the new code would suffice.

📋  **Steps to solve the problem**

 * See [CONTRIBUTING](https://github.com/einsteinpy/einsteinpy/blob/master/CONTRIBUTING.rst#development-environment) to get started with EinsteinPy development.
 * Run the Jupyter Notebooks with `einsteinpy>=0.4` (as on `master`).
 * Add, commit, push your changes
 * Submit a pull request.
 * Add this in the comments:
 	- `Addresses #<put issue number here>` if you are partially fixing the issue.
 	- `Fixes #<put issue number here>` if you are completely fixing the issue.
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by prakashaditya369**
Fixes #549
Updated the jupyter notebook for `symbolic` module.
Following notebooks where updated: 
- [https://docs.einsteinpy.org/en/latest/examples/Weyl%20Tensor%20symbolic%20calculation.html](https://docs.einsteinpy.org/en/latest/examples/Weyl%20Tensor%20symbolic%20calculation.html)
- [https://docs.einsteinpy.org/en/latest/examples/Einstein%20Tensor%20symbolic%20calculation.html](https://docs.einsteinpy.org/en/latest/examples/Einstein%20Tensor%20symbolic%20calculation.html)
- [https://docs.einsteinpy.org/en/latest/examples/Lambdify%20symbolic%20calculation.html](https://docs.einsteinpy.org/en/latest/examples/Lambdify%20symbolic%20calculation.html)
- [https://docs.einsteinpy.org/en/latest/examples/Playing%20with%20Contravariant%20and%20Covariant%20Indices%20in%20Tensors%28Symbolic%29.html](https://docs.einsteinpy.org/en/latest/examples/Playing%20with%20Contravariant%20and%20Covariant%20Indices%20in%20Tensors%28Symbolic%29.html)
- [https://docs.einsteinpy.org/en/latest/examples/Predefined%20Metrics%20in%20Symbolic%20Module.html](https://docs.einsteinpy.org/en/latest/examples/Predefined%20Metrics%20in%20Symbolic%20Module.html)
- [https://docs.einsteinpy.org/en/latest/examples/Ricci%20Tensor%20and%20Scalar%20Curvature%20symbolic%20calculation.html](https://docs.einsteinpy.org/en/latest/examples/Ricci%20Tensor%20and%20Scalar%20Curvature%20symbolic%20calculation.html)
- [https://docs.einsteinpy.org/en/latest/examples/Symbolically%20Understanding%20Christoffel%20Symbol%20and%20Riemann%20Curvature%20Tensor%20using%20EinsteinPy.html](https://docs.einsteinpy.org/en/latest/examples/Symbolically%20Understanding%20Christoffel%20Symbol%20and%20Riemann%20Curvature%20Tensor%20using%20EinsteinPy.html)

@JeS24

### CODE REVIEW

**Comment by JeS24** [comment]
@prakashaditya369 There are unnecessary imports in some of the notebooks. Change the imports in these notebooks, with the ones I have listed here. Also, we should rename some of these notebooks. I have mentioned my suggestions below. If you can think of better names, please use those.

* [Einstein Tensor symbolic calculation](https://docs.einsteinpy.org/en/latest/examples/Einstein%20Tensor%20symbolic%20calculation.html) -> **"Symbolic Module: Einstein Tensor"**:
```
import sympy
from sympy import symbols, sin, cos, sinh
from einsteinpy.symbolic import EinsteinTensor, MetricTensor
```
* [Lambdify in symbolic module](https://docs.einsteinpy.org/en/latest/examples/Lambdify%20symbolic%20calculation.html) -> **"Symbolic Module: Lambdify"**:
```
import sympy
from sympy import symbols
from einsteinpy.symbolic import BaseRelativityTensor
```
* [Playing with Contravariant and Covariant Indices in Tensors(Symbolic)](https://docs.einsteinpy.org/en/latest/examples/Playing%20with%20Contravariant%20and%20Covariant%20Indices%20in%20Tensors%28Symbolic%29.html) -> **"Symbolic Module: Tensor Indexing"**:
```
import sympy
from einsteinpy.symbolic import ChristoffelSymbols, RiemannCurvatureTensor
from einsteinpy.symbolic.predefined import Schwarzschild
```
* [Predefined Metrics in Symbolic Module](https://docs.einsteinpy.org/en/latest/examples/Predefined%20Metrics%20in%20Symbolic%20Module.html) -> **"Symbolic Module: Predefined Metrics"**:
```
import sympy
from sympy import simplify
from einsteinpy.symbolic import RicciScalar
from einsteinpy.symbolic.predefined import Schwarzschild, DeSitter, AntiDeSitter, Minkowski, find
```
* [Ricci Tensor and Scalar Curvature symbolic calculation](https://docs.einsteinpy.org/en/latest/examples/Ricci%20Tensor%20and%20Scalar%20Curvature%20symbolic%20calculation.html) -> **"Symbolic Module: Ricci Tensor and Scalar Curvature"**:
```
import sympy
from einsteinpy.symbolic import RicciTensor, RicciScalar
from einsteinpy.symbolic.predefined import AntiDeSitter
```
* [Symbolically Understanding Christoffel Symbol and Riemann Curvature Tensor using EinsteinPy](https://docs.einsteinpy.org/en/latest/examples/Symbolically%20Understanding%20Christoffel%20Symbol%20and%20Riemann%20Curvature%20Tensor%20using%20EinsteinPy.html) -> **"Symbolic Module: Christoffel Symbols and Riemann Curvature Tensor"**.
* [Weyl Tensor calculations using Symbolic module](https://docs.einsteinpy.org/en/latest/examples/Weyl%20Tensor%20symbolic%20calculation.html) -> **"Symbolic Module: Weyl Tensor"**.

Ensure a blank line between the imports and `sympy.init_printing()`.

**Comment by prakashaditya369** [comment]
@JeS24 Should I also change the headings of the Jupyter notebooks or only rename it?

**Comment by JeS24** [comment]
@prakashaditya369 Just rename them.

**Comment by JeS24** [comment]
@prakashaditya369 I made a mistake. Please replace all `:` with `-` in the names (with 1 blank space on each side of `-`). Windows does not allow `:` in path. You can amend to this commit to avoid making a fresh commit. If you don't know how, [this](https://github.com/einsteinpy/einsteinpy/wiki/Development-tips#after-you-push-proceed-with-caution) will help. It's just 3 commands.

**Comment by JeS24** [comment]
@prakashaditya369 Also add a CHANGELOG entry and your GitHub name under "CONTRIBUTORS" [here](https://github.com/qbiwan/einsteinpy/blob/master/CHANGELOG).

**Comment by JeS24** [comment]
You need to change these filenames [here](https://github.com/prakashaditya369/einsteinpy/blob/549/Updated_symbolic_jupyter_notebooks/docs/source/jupyter.rst) as well:
image

File: docs/source/jupyter.rst [hunk 1]
```diff
@@ -27,11 +27,11 @@ Jupyter notebooks
...
-   /examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb
-   /examples/Symbolically Understanding Christoffel Symbol and Riemann Curvature Tensor using EinsteinPy.ipynb
-   /examples/Weyl Tensor symbolic calculation.ipynb
+   /examples/Symbolic Module - Einstein Tensor.ipynb
+   /examples/Symbolic Module - Lambdify.ipynb
+   /examples/Symbolic Module - Tensor Indexing.ipynb
+   /examples/Symbolic Module - Predefined Metrics.ipynb
+   /examples/Symbolic Module - Ricci Tensor and Scalar Curvature.ipynb
+   /examples/Symbolic Module - Christoffel Symbols and Riemann Curvature Tensor.ipynb
+   /examples/Symbolic Module - Weyl Tensor.ipynb
```
Comment (JeS24) [inline docs/source/jupyter.rst:37]: @prakashaditya369 Leave only 1 blank line at the end.

Comment (prakashaditya369) [reply docs/source/jupyter.rst:37]: @JeS24  I think there is only 1 blank line at the end.

Comment (JeS24) [reply docs/source/jupyter.rst:37]: image

**Comment by JeS24** [comment]
@prakashaditya369 Edit the CHANGELOG and remove 1 blank line in `jupyter.rst` and we can merge.

File: docs/source/jupyter.rst (hunk 1 above)
Comment (prakashaditya369) [reply docs/source/jupyter.rst:37]: Actually this isn't showing in my text editor, so I updated it from github only.

**Comment by JeS24** [comment]
@prakashaditya369 This is weird. I am not sure, why I have access to your branch. There are 5 redundant commits in this PR (not counting the `amend`s) and the extra blank line is still there in that file.

**Comment by JeS24** [comment]
Can you try `git reset` or `rebase`?

**Comment by JeS24** [comment]
@prakashaditya369 Backup the changes first.

**Comment by prakashaditya369** [comment]
@JeS24 What options should I use with `git reset`? I hadn't used it before and it's documentation seems risky.

File: docs/source/jupyter.rst (hunk 1 above)
Comment (JeS24) [reply docs/source/jupyter.rst:37]: > Actually this isn't showing in my text editor, so I updated it from github only.
 Use a "code" editor. Perhaps, your text editor is interpreting that last blank line as EOF.

**Comment by JeS24** [comment]
@prakashaditya369 First, create a backup of the changed files. Then try this: https://stackoverflow.com/a/3293592/11922029. The new `git switch` syntax is easier to use and understand.

If it seems too daunting, I'd recommend creating another branch, copying over the changes and opening another PR. Also, do not use `/` in branch names, unless with something like a WIP PR.

**Comment by prakashaditya369** [comment]
Okay, I am creating a new branch.

**Comment by prakashaditya369** [comment]
@JeS24  Created pull request #566

**Comment by JeS24** [comment]
Moved to #566. Closing this.

**Pull request closed without merging**